packages/webapp/pages/welcome/index.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { ReactElement, useEffect } from 'react';
+import Feed, { FeedProps } from '@dailydotdev/shared/src/components/Feed';
+import { ANONYMOUS_FEED_QUERY } from '@dailydotdev/shared/src/graphql/feed';
+import {
+  OtherFeedPage,
+  generateQueryKey,
+} from '@dailydotdev/shared/src/lib/query';
+import { useAuthContext } from '@dailydotdev/shared/src/contexts/AuthContext';
+import { ViewSize, useViewSize } from '@dailydotdev/shared/src/hooks';
+import { useRouter } from 'next/router';
+import {
+  onboardingUrl,
+  webappUrl,
+} from '@dailydotdev/shared/src/lib/constants';
+import Logo, { LogoPosition } from '@dailydotdev/shared/src/components/Logo';
+import {
+  Button,
+  ButtonSize,
+  ButtonVariant,
+} from '@dailydotdev/shared/src/components/buttons/Button';
+import { AuthTriggers } from '@dailydotdev/shared/src/lib/auth';
+import { cloudinary } from '@dailydotdev/shared/src/lib/image';
+import classNames from 'classnames';
+import { NextSeo, NextSeoProps } from 'next-seo';
+import { authGradientBg } from '@dailydotdev/shared/src/components/auth';
+import { getLayout as getFooterNavBarLayout } from '../../components/layouts/FooterNavBarLayout';
+import { getLayout } from '../../components/layouts/FeedLayout';
+import { defaultOpenGraph, defaultSeo, defaultSeoTitle } from '../../next-seo';
+
+const seo: NextSeoProps = {
+  title: defaultSeoTitle,
+  openGraph: { ...defaultOpenGraph },
+  ...defaultSeo,
+};
+
+const DemoPage = (): ReactElement => {
+  const router = useRouter();
+  const { user, showLogin, isAuthReady, isLoggedIn } = useAuthContext();
+  const isLaptop = useViewSize(ViewSize.Laptop);
+
+  const feedProps: FeedProps<void> = {
+    feedName: OtherFeedPage.Welcome,
+    feedQueryKey: generateQueryKey(OtherFeedPage.Welcome, user),
+    query: ANONYMOUS_FEED_QUERY,
+    pageSize: 10,
+    allowFetchMore: false,
+  };
+
+  useEffect(() => {
+    if (isLaptop) {
+      router.replace(onboardingUrl);
+    }
```

Comment:
It's mobile + tablet or mobile only?

Reply:
I've put it on tablet as well since sone phones might be large, page also works on desktop actually but I think this is ok.